### PR TITLE
Large File Optimization

### PR DESCRIPTION
**Pull request by hpractv**
Large zip file transfer was a lot slower using get and set content.

### CODE REVIEW

File: deploy_windows_files.ps1 [hunk 1]
```diff
@@ -49,14 +49,20 @@ if ($clean_deployment_folder) {
     Invoke-RemoteCommand -Command $clean -Arguments $deployment_folder_path
 }
 
-[Byte[]]$zip = Get-Content -Path $source_zip_file_path -Encoding Byte
-$zip_size = (Get-Item -Path $source_zip_file_path).Length / 1KB
+Write-Output "Copy file: $source_zip_file_path"
+
+$copy_session = New-PSSession $server -SessionOption $so -UseSSL -Credential $credential
```
Comment (JosephDSchwartz) [inline deploy_windows_files.ps1]: If we're setting up this session, is there a reason to not use it for the other remote calls?

File: deploy_windows_files.ps1 [hunk 2]
```diff
@@ -49,14 +49,20 @@ if ($clean_deployment_folder) {
...
 
 $copy = {
-    param([string]$path, [string]$file, [Byte[]]$zip_data, [int]$file_size)
-    Write-Host "Writing Package Archive: $file"
-    Write-Host "File Size: $file_size KB"
-    Set-Content -Path $file -Value $zip_data -Encoding Byte
+    param([string]$path, [string]$file) #, [Byte[]]$zip_data, [int]$file_size)
+    # Write-Host "Writing Package Archive: $file"
+    # Write-Host "File Size: $file_size KB"
+    # Set-Content -Path $file -Value $zip_data -Encoding Byte
```
Comment (JosephDSchwartz) [inline deploy_windows_files.ps1]: Did you want to keep all of these comments? If so, do you think a comment would be good to explain why (so others don't have my same question)?

Comment (hpractv) [reply deploy_windows_files.ps1]: I forgot to check-in my comments clean up...should be better now.

**Review by JosephDSchwartz — APPROVED** [review]
These latest changes really seemed to clean things up!

**Comment by hpractv** [comment]
> These latest changes really seemed to clean things up!

Thanks! The code is a lot cleaner using `Copy-Item` instead of trying to read the file all into memory and the push it across `Invoke-Command`.

**Pull request merged**